polybrain.el

```diff
@@ -135,7 +131,7 @@ changed."
 
 (define-innermode poly-brain-org-innermode
   :mode 'org-mode
-  :head-matcher "--- Entry.*"
+  :head-matcher "^[─-]\\{3\\} Entry [─-]+"
```

Comment:
I use `ascii-art-to-unicode` which replaces the `-` with other characters. This regex should match them.